kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ProducerMetrics.java

```diff
@@ -50,339 +40,226 @@
-        .addRate(
-            ProducerMetricsRegistry.RESPONSE_SEND_RATE,
-            ProducerMetricsRegistry.RESPONSE_SEND_RATE_DOC)
-        .addWindowedCount(
-            ProducerMetricsRegistry.RESPONSE_COUNT_WINDOWED,
-            ProducerMetricsRegistry.RESPONSE_COUNT_WINDOWED_DOC)
-        .build();
-
-    new SensorBuilder(metricMBean, jmxPrefix, ProducerMetricsRegistry.RECORD_ERROR_SENSOR)
-        .addRate(
-            ProducerMetricsRegistry.RECORD_ERROR_RATE,
-            ProducerMetricsRegistry.RECORD_ERROR_RATE_DOC)
-        .addWindowedCount(
-            ProducerMetricsRegistry.ERROR_COUNT_WINDOWED,
-            ProducerMetricsRegistry.ERROR_COUNT_WINDOWED_DOC)
-        .build();
-
-    new SensorBuilder(metricMBean, jmxPrefix, ProducerMetricsRegistry.REQUEST_LATENCY_SENSOR)
-        .addMax(
-            ProducerMetricsRegistry.REQUEST_LATENCY_MAX,
-            ProducerMetricsRegistry.REQUEST_LATENCY_MAX_DOC)
-        .addAvg(
-            ProducerMetricsRegistry.REQUEST_LATENCY_AVG,
-            ProducerMetricsRegistry.REQUEST_LATENCY_AVG_DOC)
-        .addPercentiles(
-            ProducerMetricsRegistry.REQUEST_LATENCY_PCT,
-            ImmutableMap.of(
-                "p95", 0.95,
-                "p99", 0.99,
-                "p999", 0.999),
-            ProducerMetricsRegistry.REQUEST_LATENCY_PCT_DOC)
-        .build();
-    log.info("Successfully registered kafka-rest produce metrics with JMX");
-  }
+  // response
+  static final String RESPONSE_SEND_RATE_METRIC_NAME = "response-rate";
+  private static final String RESPONSE_SEND_RATE_METRIC_DOC =
+      "The average number of responses sent per second.";
 
-  private static final class BeanCoordinate {
-    final String beanName;
-    final Map<String, String> tags;
+  static final String RESPONSE_COUNT_WINDOWED_METRIC_NAME = "response-count-windowed";
+  private static final String RESPONSE_COUNT_WINDOWED_METRIC_DOC =
+      "The total number of responses sent in the given window.";
 
-    BeanCoordinate(String beanName, Map<String, String> tags) {
-      this.beanName = requireNonNull(beanName);
-      this.tags = ImmutableMap.copyOf(tags);
-    }
+  // errors
+  static final String ERROR_COUNT_WINDOWED_METRIC_NAME = "error-count-windowed";
+  private static final String ERROR_COUNT_WINDOWED_METRIC_DOC =
+      "The total number of record sends that resulted in errors in the given window.";
 
-    @Override
-    public boolean equals(Object o) {
-      if (this == o) {
-        return true;
-      }
-      if (!(o instanceof BeanCoordinate)) {
-        return false;
-      }
-      BeanCoordinate that = (BeanCoordinate) o;
-      return Objects.equals(beanName, that.beanName) && Objects.equals(tags, that.tags);
-    }
+  static final String RECORD_ERROR_RATE_METRIC_NAME = "record-error-rate";
+  private static final String RECORD_ERROR_RATE_METRIC_DOC =
+      "The average per-second number of record sends that resulted in errors.";
```

Comment:
nit: These sound way too different while in fact they seem to be rate and count metrics for the same event type (produce send errors). I'd suggest prepending `ERROR_COUNT_WINDOWED...` with `RECORD_`.

Reply:
Done.